rhel/7.7/aws/DCOS-1.13.5/docker-18.09.2-ce/selinux_disabled/install_dcos_prerequisites.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+sudo setenforce 0 && \
+sudo sed -i --follow-symlinks 's/^SELINUX=.*/SELINUX=disabled/g' /etc/sysconfig/selinux
+
+sudo sed -i '$ d' /etc/resolv.conf
+sudo bash -c 'echo -e "nameserver 8.8.8.8\n" >> /etc/resolv.conf'
+
+sudo yum install -y yum-utils
+
+sudo yum-config-manager --add-repo https://download.docker.com/linux/centos/docker-ce.repo
+sudo yum makecache fast
+
+sudo yum install -y http://mirror.centos.org/centos/7/extras/x86_64/Packages/container-selinux-2.74-1.el7.noarch.rpm
```

Comment:
Version 2.74-1 of container-selinux package has been updated to 2.107-1: http://mirror.centos.org/centos/7/extras/x86_64/Packages/

Please change the above command to:
sudo yum install -y http://mirror.centos.org/centos/7/extras/x86_64/Packages/container-selinux-2.107-1.el7_6.noarch.rpm